book/src/simple-payment-and-state-verification.md

```diff
@@ -0,0 +1,158 @@
+transaction to the required set of validator votes.
+
+An entry hash is a merkle tree of all the transactions sorted by signature.
+
+
+```
+                          Entry hash
+               /                           \
+            hash                             hash
+     /             \                      /       \
+    /               \                    /         \
+hash(T1, status)  hash(T2, status)   hash(T3, status)  0
+
+```
+
+Transaction status is necessary for the receipt because the state receipt is
+constructed for the block.  Two transactions over the same state can appear in
+the block, and therefore, there is no way to infer from just the state whether a
+transaction that is encoded in the ledger actually succeeded in a state commit
+or failed.  It may not be necessary to encode the full code but a single
+status bit to indicate the transaction succeeded.
+
+```
+Entry {
+    poh: Hash,
+    num_hashes: uint64,
+    success: Vec<Transaction>
+    program_errors: Vec<Transaction>
+}
+```
+
+### State Merkle Path
+
+A state receipt provides a receipt that a specific state is committed at 
+the end of the block.  Inter-block state transitions do not generate a receipt.
+
+For example:
+
+* A sends 5 lamports to B
+* B spends 5 lamports
+* C sends 5 lamports to A
+
+At the end of the block, A and B are in the exact same starting state, and any
+state receipt would point to the same value for A or B.
+
+ 
+```
+                          Bank hash
+               /                           \
+            hash                             previous bank hash
+     /             \                    
+    /               \                   
+hash(Account1)  hash(Account2) 
+
+```
+
+A state receipt contains only the updated state changes for the block.  A direct
+merkle path to the current bank hash guarantees the state value at that bank
+hash, but it cannot be used to generate a “current” receipt to the latest state
+if the state modification occurred in some previous block.  There is no
+guarantee that the path provided by the validator is the latest one available
+out of all the previous bank hashes.
+
+
+Clients that want to query the chain for a receipt of the "latest" state would
+need to create a transaction that would update the merkle path for that account,
+such as a credit of 0 lamports.
+
+###  Validator Votes
+
+Leaders should coalesce the validator votes by stake weight into a single entry.
+
+###  Chain of Entries
+
+A receipt is a PoH link from the payment or state merkle path to a consecutive
+list of validation votes.
+
+
+It contains the following:
+* State -> Bank Hash
+or 
+* Transaction -> Entry Hash
+
+And a vector of PoH entries:
+
+* Validator vote entries
+```
+// full entry with transactions
+Entry {
+    poh: Hash,
+    num_hashes: uint64,
+    success: Vec<Transaction>
+    //should be empty
+    program_errors: Vec<Transaction>
+}
+```
+* Tick's
+* Compressed entries 
+
+```
+CompressedEntry { 
```

Comment:
If you take the LightClient suggestion, would it make sense to call this a LightEntry?  I'd imagine we may want to one day compress either an Entry or LightEntry.